aptos-move/framework/aptos-framework/sources/vesting.spec.move

```diff
@@ -45,8 +45,16 @@ spec aptos_framework::vesting {
     }
 
     spec total_accumulated_rewards(vesting_contract_address: address): u64 {
-        // TODO: Verification out of resources/timeout
-        pragma verify = false;
+        pragma verify_duration_estimate = 120;
+
+        include TotalAccumulatedRewardsAbortsIf;
+    }
+
+    spec schema TotalAccumulatedRewardsAbortsIf {
+        vesting_contract_address: address;
+
+        requires staking_contract.commission_percentage >= 0 && staking_contract.commission_percentage <= 100;
```

Comment:
Could you briefly explain why we need this requirement here?

Reply:
I add comment there